lib/integrations/calendar/services/Office365CalendarService.ts

```diff
@@ -0,0 +1,249 @@
+import { Calendar as OfficeCalendar } from "@microsoft/microsoft-graph-types-beta";
+import { Credential } from "@prisma/client";
+
+import { getLocation, getRichDescription } from "@lib/CalEventParser";
+import { handleErrorsJson, handleErrorsRaw } from "@lib/errors";
+import { CALENDAR_INTEGRATIONS_TYPES } from "@lib/integrations/calendar/constants/generals";
+import logger from "@lib/logger";
+import prisma from "@lib/prisma";
+
+import { BatchResponse, EventBusyDate, NewCalendarEventType } from "../constants/types";
+import { Calendar, CalendarEvent, IntegrationCalendar } from "../interfaces/Calendar";
+import { BufferedBusyTime, O365AuthCredentials } from "../interfaces/Office365Calendar";
+
+const { MS_GRAPH_CLIENT_ID = "", MS_GRAPH_CLIENT_SECRET = "" } = process.env;
```

Comment:
I also prefer your style, but this is a webpack plugin limitation. 😞 
```suggestion
const MS_GRAPH_CLIENT_ID = process.env.MS_GRAPH_CLIENT_ID || "";
const MS_GRAPH_CLIENT_SECRET = process.env.MS_GRAPH_CLIENT_SECRET || "";
```